src/Apps/Home/Components/HomeContentCards.tsx

```diff
@@ -51,15 +115,16 @@ export const HomeContentCards: React.FC = () => {
     })
   }, [])
 
-  if (cards.length < 1) {
-    return <h1>placeholder</h1>
-  }
+  const placeholderCards = [
+    <HomeContentCardPlaceholder />,
+    <HomeContentCardPlaceholder />,
+  ]
 
-  return (
-    <HeroCarousel>
-      {cards.map((card, index) => (
-        <ContentCard card={card} key={card.id} index={index} />
-      ))}
-    </HeroCarousel>
-  )
+  const realCards = cards.map((card, index) => (
+    <ContentCard card={card} key={card.id} index={index} />
+  ))
+
+  const heroCards = cards.length < 1 ? placeholderCards : realCards
```

Comment:
What we typically do is write a placeholder for the entire component and then wrap it in `Skeleton` which performs that pulsing animation and disables interaction.

Reply:
Ok update, we chatted about this here:

https://artsy.slack.com/archives/C01ADJNCS5D/p1668542811835079?thread_ts=1668525827.325359&cid=C01ADJNCS5D

And ended up bailing on the idea of the Skeleton wrapper because of the wrinkle that this is in the context of a full bleed component and it's not behaving as we'd like. Can come back to this if we choose to!